### PR TITLE
fixed forgotten password tests

### PR DESCRIPTION
**Pull request by qaga**
@justnpT

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java [hunk 1]
```diff
@@ -36,5 +44,16 @@ public SignInPage verifyErrorMessage(String errorMessage) {
...
     }
+
+    public void login(String username, String password) {
+        usernameField.sendKeys(username);
+        passwordField.sendKeys(password);
+        signInButton.click();
+    }
+
+    public SignInPage clickForgotPasswordLink() {
+        forgottenPasswordLink.click();
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java]: wait for element clickable before click

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java [hunk 2]
```diff
@@ -36,5 +44,16 @@ public SignInPage verifyErrorMessage(String errorMessage) {
 
         return this;
     }
+
+    public void login(String username, String password) {
+        usernameField.sendKeys(username);
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java]: wait for the elements before interaction

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java [hunk 3]
```diff
@@ -36,5 +44,16 @@ public SignInPage verifyErrorMessage(String errorMessage) {
...
+        passwordField.sendKeys(password);
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java]: wait for the element visible

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java [hunk 4]
```diff
@@ -36,5 +44,16 @@ public SignInPage verifyErrorMessage(String errorMessage) {
...
+        signInButton.click();
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java]: wait for the element clickable

File: src/test/java/com/wikia/webdriver/elements/mercury/pages/login/RegisterPage.java [hunk 5]
```diff
@@ -2,20 +2,21 @@
...
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
-public class JoinPage extends WikiBasePageObject {
+public class RegisterPage extends WikiBasePageObject {
 
     @FindBy(css = ".footer-callout-emphasis")
     private WebElement SignInButton;
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/pages/login/RegisterPage.java]: MAJOR Rename this field "SignInButton" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MAJOR 1 major

Watch the comments in this conversation to review them.

**Pull request merged**